app-web/gatsby/createSchemaCustomization.js

```diff
@@ -13,12 +13,16 @@ module.exports = ({ actions }) => {
       path: String
       id: String! 
     }
+    type frontmatterLableSet {
+      app: String
+      importance: String
```

Comment:
I think we should leave importance out for now as there is no clear definition for it.